pkg/chunk/aws/dynamodb_table_client.go

```diff
@@ -264,6 +269,18 @@ func (d dynamoTableClient) DescribeTable(ctx context.Context, name string) (desc
 	return
 }
 
+// Filter out errors that we don't want to see
+// (currently only relevant in integration tests)
+func relevantError(err error) bool {
+	if err == nil {
+		return false
+	}
+	if strings.Contains(err.Error(), "Tagging is not currently supported in DynamoDB Local.") {
```

Comment:
Is it possible to do this without string matching?  Is there a `Code()` for it like here:https://github.com/cortexproject/cortex/blob/master/pkg/chunk/aws/dynamodb_storage_client.go#L251

Reply:
Sadly `Code()` is `"UnknownOperationException"` in this case.
I can't see a repo or similar to file issues against DynamoDB local.